dhall/src/Dhall/Parser/Expression.hs

```diff
@@ -133,18 +140,29 @@ parsers embedded = Parsers {..}
 
         alternative2 = do
             let binding = do
-                    _let
+                    _let_
+
+                    src0 <- note nonemptyWhitespace
```

Comment:
Is it a convention to include comments in the term "whitespace"? I wonder whether there's a better name…

Reply:
Yeah, I had the same thought, too.  I'm not sure what terminology other languages use to distinguish comment-free whitespace from potentially-commented whitespace.

I can speak to why the standard grammar uses the term "whitespace" to include comments, which is because the parsing tutorials I used when writing the language followed that convention.